index.html

```diff
@@ -285,107 +271,211 @@ <h2>
-        there is no battery attached to the system, or otherwise. When the
-        battery discharging time is updated, the <a>user agent</a> MUST
-        <a>queue a task</a> which sets the <code>dischargingTime</code>
-        attribute's value and <a>fires an event</a> named
-        <code><a>dischargingtimechange</a></code> at the <a>BatteryManager</a>
-        object.
-      </p>
-      <p>
-        The <code id="widl-BatteryManager-level">level</code> attribute MUST be
-        set to 0 if the system's battery is depleted and the system is about to
-        be suspended, and to 1.0 if the battery is full, the implementation is
-        unable to report the battery's level, or there is no battery attached
-        to the system. When the battery level is updated, the <a>user agent</a>
-        MUST <a>queue a task</a> which sets the <code>level</code> attribute's
-        value and <a>fires an event</a> named <code><a>levelchange</a></code>
-        at the <a>BatteryManager</a> object.
-      </p>
-      <p class="note">
-        The definition of how often the <code><a>chargingtimechange</a></code>,
-        <code><a>dischargingtimechange</a></code>, and
-        <code><a>levelchange</a></code> events are fired is left to the
-        implementation.
-      </p>
-      <section id="multiple-batteries">
-        <h2>
-          Multiple batteries
-        </h2>
+      <aside class="note">
+        If the user agent is <a>unable to report the battery status
+        information</a>, the <a>BatteryManager</a>'s internal slots will remain
+        with their default values, which emulate a fully charged and plugged in
+        battery. This is aimed at reducing the potential for fingerprinting and
+        preventing applications from degrading performance, if the battery
+        status information is not made available, for example.
+      </aside>
+      <section>
+        <h3>
+          Internal slots
+        </h3>
         <p>
-          If a hosting device contains more than one battery,
-          <a>BatteryManager</a> SHOULD expose an unified view of the batteries.
+          {{BatteryManager}} instances are created with the following
+          <a data-cite=
+          "ECMASCRIPT#sec-object-internal-methods-and-internal-slots">internal
+          slots</a>:
         </p>
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>
+                Internal slot
+              </th>
+              <th>
+                Initial value
+              </th>
+              <th>
+                Description (non-normative)
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <dfn>[[\Charging]]</dfn>
+              </td>
+              <td>
+                `true`
+              </td>
+              <td>
+                Charging state of the system's battery.
```

Comment:
maybe this should like to the section below? 

```suggestion
                Charging state of the system's battery. See [[[#charging-internal-slot]]].
```

Same with others... alternatively. Why not move the sections below into here? Or vice versa. Seems a bit odd to have them almost defined twice.

Reply:
I've removed the non-normative description column from the table, so that the descriptions are all gathered in the section below.